source/_integrations/hive.markdown

```diff
@@ -17,11 +17,22 @@ ha_codeowners:
 ha_domain: hive
 ---
 
-The `hive` integration is the main integration to set up and integrate all supported Hive devices. Once configured with the minimum required details it will detect and add all Hive devices into Home Assistant, including support for multi-zone heating.
+This `hive` integration uses the same username and password you use on the [Hive website](https://my.hivehome.com) to configure it within Home Assistant. Once configured Home Assisatnt will detect and add all Hive devices, including support for multi-zone heating.
 
-This integration uses the  Hive website [https://my.hivehome.com](https://my.hivehome.com) credentials, you will need to use the same username and password you use on the Hive website to configure this Hive integration in Home Assistant.
+## Configuration
 
-To add your Hive devices into your Home Assistant installation, add the following to your `configuration.yaml` file:
+Menu: *Configuration* > *Integrations*
+
+Press on **Hive** and configure the integration:
+
+- Enter you Hive Username.
+- Enter you Hive Password.
+
+If you have 2 factor authentication setup on your Hive account, a following dialog will be presented asking for  your code.
+
+### Configuration via YAML
```

Comment:
When an integration moves to the UI, the YAML configuration is deprecated and should thus be removed from the documentation.